tests/scripts/all.sh

```diff
@@ -2285,6 +2288,176 @@ component_test_psa_crypto_config_reference_all_ec_algs_use_psa () {
     tests/ssl-opt.sh
 }
 
+component_test_psa_crypto_config_accel_ecc () {
```

Comment:
This was removed in the previous PR, so I think re-introducing it here was a rebasing accident.